Yoast/javascript
# Add rule for prefix-changing exception words with peng-

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a rule for words-exceptions that start with prefix peng- preceded by a first-order prefix. If the word form is found in the indicated exception list (kBeginning), the first prefix must be dropped as usual, while the following prefix _peng_- must be replaced by _k-_.
* [yoastseo] N/A Adds Indonesian word form support to the content analysis app.
* [Yoast SEO Premium] Improves the recognition of exception words in which the prefix is replaced with k- during stemming.

## Test instructions
This PR can be tested by following these steps:

* Make sure all current specs pass
* Add more examples to the specs and make sure they pass.
* To create examples of regular words,  add a first-order prefix (e.g. mem-/ke-) followed by prefix peng- and attach suffix -kan/-an/-i.
Pattern: **mem-/di- + peng- + word + -kan/-i**  
**ke- + peng- + word + -an**. 
E.g. mempengapakan -> apa, kepengusahaan -> usaha
* To create example words from exception list, take entries from the kBeginning exception list (e.g. erusi, embali, endali) and attach the affixes, see above pattern, and add "k" in the beginning of the assigned stem result. For example, mempengerusikan > kerusi, dipengembalikan > kembali,  kepengendalian > kendali
* In Indonesian Premium: In Russian Premium: use one of the aforementioned words as a keyphrase and use forms of those words in the title/text/meta description. Check that the keyphrase forms are recognized by the plugin.
* For more information on the relevant exception list, see the [accompanying premium-configuration PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/188).

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-439
